atlascope/core/models/tour.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib import admin
+from django.db import models
+from rest_framework import serializers
+
+
+class Tour(models.Model):
+    name = models.CharField(max_length=255, default='My Tour')
+    waypoints = models.ManyToManyField('Waypoint', through='TourWaypoints')
+    investigation = models.ForeignKey(
+        'Investigation',
+        on_delete=models.CASCADE,
+        related_name='tour',
+    )
+
+
+admin.site.register(Tour)
+
+
+class TourSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Tour
+        fields = '__all__'
```

Comment:
I think you'll need to a Waypoint serializer that you invoke here so that the waypoint information is nested within the serialized tour object. Otherwise you'll just get a list of integers representing each waypoint.

Reply:
resolved in 99d34aed6212af61cfb082bacedac18ec38537e0